driver/src/main/java/org/kaazing/k3po/driver/control/handler/ControlServerHandler.java

```diff
@@ -113,7 +113,7 @@ public void prepareReceived(final ChannelHandlerContext ctx, MessageEvent evt) t
             logger.debug("preparing script(s) " + scriptNames);
         }
 
-        robot = new Robot(addressFactory, bootstrapFactory);
+        robot = new Robot();
```

Comment:
Why are you no longer using the given addressFactory?  Or at least checking to see if it's set before calling the appropriate robot constructor?